### PR TITLE
Details page - [4pt]

### PR DESCRIPTION
**Pull request by sadaf-Daneshgar**
**New Features** 🥇 
-------------------------------------------
When the User selects a specific item, they can see the details page (remember to follow [design guidelines](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)).
- Should include a "Reservation" button in the same page. (Maybe try the "configure" button styling?)
image
- [Optional] Use a table to show details about our item.
image

### CODE REVIEW

File: src/components/BookDetail.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  const { name } = useParams();
+  const findBook = books.books.find((book) => book.name === name);
+
+  if (!findBook) {
+    return <h2>Book not found</h2>;
+  }
+
+  return (
+    <>
+      <div className="main">
```
Comment (Arch-Noize) [inline src/components/BookDetail.js:17]: Mind that there seems to be a overflow issue where the design does change styling when on certain thresholds, but its not being responsive by fitting inside the given screen or shrinking its size it it must do so 
image

File: src/components/BookDetail.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <div className="book-detail">
+          <div className="book-image">
+            <img className="img" src={findBook.image} alt={findBook.name} />
+          </div>
+          <div className="book-details">
+            <h3>{findBook.name}</h3>
+            <table className="styled-table">
+              <tbody>
+                <tr>
+                  <th scope="row">Author</th>
```
Comment (Arch-Noize) [inline src/components/BookDetail.js:28]: Be sure to also add the `info` property that is returned from getting books in this page!
image

**Review by Arch-Noize — CHANGES_REQUESTED** [review]
# Changes required :yellow_circle: :yellow_circle:

Well done on adding the details unto the page and making the navigation feel smooth and understandable. There are some small styling changes you should look into.

File: src/components/BookDetail.js (hunk 1 above)
Comment (sadaf-Daneshgar) [reply src/components/BookDetail.js:17]: Hello @Arch-Noize  thanks for your nice review but see for me it show like this

**Review by Arch-Noize — APPROVED** [review]
# Approved! :green_circle: :green_circle: :green_circle: 

Approved

Well done on fixing the errors and adding the changes. There's nothing left but to merge!

**Pull request merged**